### ISSUE TITLE
First Run Issue

### ISSUE THREAD

**Issue by xuancongwen** [simoleone/git-coauthor#1]
Not the error with `cat`:

```
➜  ocb-web git:(master) git coauthor
Usage
  git coauthor
      display current coauthors and this usage message

  git coauthor <initials or name> ...
      set current co-authors to developers with matching initials or names

  git coauthor --solo
      remove current co-authors and go solo

  git coauthor --add "Jane Smith <jane@example.com>"
      add a new author to the database

  git coauthor --ls
      list all authors in the database

  git coauthor --rm <initials or name>
      remove author with matching initials or name from the database

*cat: /Users/xuancongwen/.git_coauthor_commit_template: No such file or directory*
CURRENT CO-AUTHORS
You're committing solo!
```

**Comment by simoleone** [comment]
Error is harmless functionality-wise but this needs to be polished, thanks for the report.

**Comment by simoleone** [comment]
fixed in [v1.1](https://github.com/simoleone/git-coauthor/releases/tag/v1.1). You can `brew upgrade git-coauthor` to get it!
